distribution/docs/src/main/resources/IntContents.adoc

```diff
@@ -1,10 +1,245 @@
 
-== Overview
-Distributed Data Framework ({branding}) is an agile and modular integration framework.
-It is primarily focused on data integration, enabling clients to insert, query and transform information from disparate data sources via the {branding} Catalog. A Catalog API allows integrators to insert new capabilities at various stages throughout each operation.
-{branding} is designed with the following architectural qualities to benefit integrators.
+This section supports integrating {branding} with existing applications or frameworks.
 
-This page supports integrating {branding} with existing applications or frameworks. 
+=== Quick Start
+This quick tutorial will demonstrate:
 
+- [*] Installation
+- [*] Catalog Capabilities: Ingest and query using every endpoint
+- [*] Use of the Content Framework
+- [*] Metrics Reporting
 
+==== Install {branding}
 
+. Install {branding} by unzipping the zip file.
+This will create an installation directory, which is typically created with the name and version of the application.
+This installation directory will be referred to as `<DISTRIBUTION_INSTALL_DIR>`.
+Substitute the actual directory name in place of this.
+. Start {branding} by running the `<DISTRIBUTION_INSTALL_DIR>/bin/ddf` script (or `ddf.bat` on Windows).
+. Verify the distribution is running.
+. Go to https://localhost:8993/admin.
+. Enter the default username of "admin" (no quotes) and the password of "admin" (no quotes).
+. Follow the install instructions for more extensive install guidance, or use the command line console (which appears after the  <DISTRIBUTION_INSTALL_DIR>/bin/ddf script starts) to install a few applications as mentioned below.
++
+----
+app:start catalog-app
+app:start content-app
+app:start solr-app
+----
++
+[NOTE]
+====
+Other applications may be installed at a later time.
+====
++
+. After the installation has been configured the instance should be restarted.
+. Go to https://localhost:8993/services and verify five REST services are available: admin, application, metrics, catalog, and catalog/query.
+. Click on the links to each REST service's WADL to see its interface.
+. In the Admin Console (at /admin), configure the system settings.
+.. Enter the username of "admin" (no quotes) and the password "admin" (no quotes).
+.. Select Platform app
+.. Select Platform Global Configuration.
```

Comment:
The Platform Global Configuration no longer exists. This is configured in the installer now or through the properties file. I believe there is a ticket out to  make this configurable via the admin ui including the restart.